Tasks/Common/npm-common/npmrcparser.ts

```diff
@@ -21,10 +20,9 @@ export function GetRegistries(npmrc: string): string[] {
     return registries;
 }
 
-export function NormalizeRegistry(registry: string): string {
-    if (registry) {
-        registry = registry.slice(-1) !== '/' ? registry + '/' : registry;
+export function NormalizeRegistry(registry) {
```

Comment:
nit: add type information on parameter and return type

Reply:
Done